HW_7/view.py

```diff
@@ -0,0 +1,73 @@
+from logger import log
+
+
+@log
+def greatings():
+    '''Приветствие.'''
+    print('Справочник')
+
+
+@log
+def menu() -> int:
+    """
+    Вывод меню.
+    """
+    print('Меню')
+    print(
+        '0 - Выход \n1 - Загрузить из файла и вывести на экран \n2 - Добавить новую запись \n3 - Редактировать запись по id \n')
+    return int(input("Введите пункт меню: "))
```

Comment:
Можно все поместить внутрь функции input
```suggestion
    return int(input(
        'Меню\n'
        '0 - Выход\n'
        '1 - Загрузить из файла и вывести на экран\n'
        '2 - Добавить новую запись\n'
        '3 - Редактировать запись по id\n'
        'Введите пункт меню: '
    ))
```